packages/terra-form-select/translations/en-US.json

```diff
@@ -17,5 +17,8 @@
   "Terra.form.select.multiSelectUsageGuidance": "Enter text or use up and down arrow keys to navigate through options. Press enter to select or unselect an option.",
   "Terra.form.select.searchUsageGuidance": "Enter text or use up and down arrow keys to navigate through options. Press enter to select an option.",
   "Terra.form.select.clearSelect": "Clear select",
-  "Terra.form.select.selectCleared": "Select value cleared"
+  "Terra.form.select.selectCleared": "Select value cleared",
+  "Terra.form.select.display": "display",
+  "Terra.form.select.menu": "menu",
```

Comment:
We want to get translations for these values for the supported locales

Reply:
I have logged Jira for it.